peer/brontide.go

```diff
@@ -592,6 +593,16 @@ func (p *Brontide) Start() error {
 	p.log.Debugf("Loaded %v active channels from database",
 		len(activeChans))
 
+	// Subscribe channel events before loading channels so we won't miss
+	// events.
+	//
+	// NOTE: ChannelNotifier must be started before subscribing events
+	// otherwise we'd panic here.
+	sub, err := p.cfg.ChannelNotifier.SubscribeChannelEvents()
```

Comment:
I think the fix will work as intended, but the reenableTimeout is 19 minutes by default. I'm wondering if it's possible that we receive many channel events before then that stack up in the notifier's overflow queue and cause a DoS. Maybe this could happen if many peers have flapping connections and multiple links each.

Reply:
Maybe we could do a check on `reenableTimeout`, if it's less than, say 2 min, then we make this subscription, otherwise we won't bother retrying enabling the channel again?